app/controllers/short_url_requests_controller.rb

```diff
@@ -8,6 +8,7 @@ def index
   end
 
   def show
+    @existing_redirect = Redirect.where(from_path: @short_url_request.from_path).first
```

Comment:
There can be multiple `ShortUrlRequests`, but there can only be a single `Redirect` for a given `from_path` -- which is created or updated when a request is accepted.

See https://github.com/alphagov/short-url-manager/blob/master/app/models/short_url_request.rb#L28-L42